generator/graphql-kotlin-schema-generator/src/test/kotlin/com/expediagroup/graphql/generator/test/integration/OptionalInputTest.kt

```diff
@@ -46,7 +46,7 @@ class OptionalInputTest {
             Arguments.of("{ optionaListScalarInput(input: [\"ABC\"] )  }", "input value: [ABC]"),
             Arguments.of("{ optionaListInputObject }", "input was not specified"),
             Arguments.of("{ optionaListInputObject(input: null) }", "input value: null"),
-            Arguments.of("{ optionaListInputObject(input: [{id: 1, name: \"ABC\"}] )  }", "input value: [{id=1, name=ABC}]"),
+            Arguments.of("{ optionaListInputObject(input: [{id: 1, name: \"ABC\"}] )  }", "input value: [SimpleArgument(id=1, name=ABC)]"),
```

Comment:
We are now properly passing the wrapped class info down so the `toString` changed